modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-ack.adoc

```diff
@@ -0,0 +1,153 @@
+= Anypoint MQ ACK and NACK Operations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+== Acknowledgment Configurations
+
+The Anypoint MQ connector provides three acknowledgment configurations: Immediate, Automatic, and Manual.
+
+=== Immediate Acknowledgment
+
+To acknowledge messages as your app consumes them, before processing them, set the `acknowledgementMode` to `IMMEDIATE` either in the Subscriber or Consume operations.
+
+Acknowledging the message automatically when it's received means that the message isn't redelivered if an error occurs during message processing. To handle errors without losing messages, use a dead letter queue.
+
+If an error occurs while trying to acknowledge the message using `IMMEDIATE` mode, an `ANYPOINT-MQ:ACKING` error is thrown.
+
+=== Automatic Acknowledgment
+
+To acknowledge a received message automatically only if the flow execution completes successfully, set the `acknowledgementMode` to `AUTO` in the Subscriber operation.
+
+If an error occurs during the flow execution, causing its execution to terminate prematurely, the message is NACKed and returns to the queue for redelivery.
```

Comment:
"...the message is NACKed and returns...."  >  "the message is not acknowledged ('NACK') and returns...."

GLOBALLY reword all syntax used as narrative nouns and verbs.